src/sentry/integrations/slack/endpoints/action.py

```diff
@@ -42,6 +50,7 @@
 NO_ACCESS_MESSAGE = "You do not have access to the organization for the invitation."
 NO_PERMISSION_MESSAGE = "You do not have permission to approve member invitations."
 NO_IDENTITY_MESSAGE = "Identity not linked for user."
+ENABLE_SLACK_SUCCESS_MESSAGE = "Slack has been enabled."
```

Comment:
Maybe this could be like "Slack notifications have been enabled" because as it is, it reads a bit strangely since the user obviously came from Slack.